src/main/scala/com/github/xiaodongw/swagger/finatra/SwaggerController.scala

```diff
@@ -6,17 +6,21 @@ import com.twitter.finatra.response.Mustache
 import io.swagger.models.Swagger
 import io.swagger.util.Json
 
-@Mustache("index")
-case class SwaggerView(title: String, path: String)
-
 class SwaggerController(docPath: String = "/api-docs", swagger: Swagger) extends Controller {
+
   get(s"${docPath}/model") { request: Request =>
     response.ok.body(Json.mapper.writeValueAsString(swagger))
       .contentType("application/json").toFuture
   }
 
   get(s"${docPath}/ui") { request: Request =>
-    response.temporaryRedirect
-        .location("/webjars/swagger-ui/2.2.8/index.html?url=/api-docs/model")
+    redirectToSwaggerUi()
   }
+
+  get(s"${docPath}") { request: Request =>
+    redirectToSwaggerUi()
+  }
+
+  private def redirectToSwaggerUi() =
+    response.temporaryRedirect.location(s"/webjars/swagger-ui/2.2.8/index.html?url=${docPath}/model")
```

Comment:
I'm sorry.  I'm not sure I understand. This url is the web jars where swagger ui is hosted. Another option would be to copy the sources locally to the repo or do a git reference to swagger ui repo and then directly reply with the ui. I guess to fix https://github.com/xiaodongw/swagger-finatra/issues/14 for example this is what needs to be done.

This seems to me independent of the fact how the swagger finatra endpoints reply. They could either reply with the ui or the swagger json. The routing /docPath is not used for now. So instead of just replying nothing it could reply with the ui or the swagger document. I would prefer the ui because it is shorter, the general entry point for a user and automatic tools retrieving the swagger json don't care about the path length.
So I would like it to behave like this /api-docs, /api-docs/ui => Reply with UI, /api-docs/model => Reply with swagger json

Please correct me if I got something wrong.